assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -6,176 +6,163 @@
 
 
 {% block details%}
+    {# HTML 5 only solution where the form is outside the table, should change it sometime probably #}
+    <form method="post"
+          action="{% url 'slot_list' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}"
+          id="statsForm">
+        {% csrf_token %}
+    </form>
 
-    <form method="post">
-        <div class="row">
-            <div class="col-12 table-responsive">
-                <table class="table table-hover">
-                    {# Table header #}
-                    <thead>
-                    <tr class="">
-                        <td style="min-width: 10rem">Name</td>
-                        <td style="min-width: 15rem;">Help stats</td>
-                        {% if show_delete_column %}
-                            <td>Delete</td>
-                        {% endif %}
-                    </tr>
-                    </thead>
-
-                    <tbody>
-                    {# Table rows #}
-                    {% for entry in entries %}
-                        {% with entry=entry.get_subclass %}
-                            <tr>
-                                <td>
-                                    <div class="d-flex justify-content-between align-items-center">
-                                        <span>
-                                            {{ entry.get_name }}
-                                            {% if entry.is_external or entry.created_by != entry.user %}
-                                                <small class="font-italic">added by {{ entry.created_by }}</small>
-                                            {% endif %}
-                                        </span>
+    <div class="row">
+        <div class="col-12 table-responsive">
+            <table class="table table-hover">
+                {# Table header #}
+                <thead>
+                <tr class="">
+                    <td style="min-width: 10rem">Name</td>
+                    <td style="min-width: 15rem;">Help stats</td>
+                    <td></td>
```

Comment:
If nothing can be deleted, there is no reason to reserve space for the column. And yes, I know it won't have content and thus be smaller, But running here locally still shows a total width of 13 interior + 2*12 padding. That's why I had the can_delete_some paramater. It was a conscious styling choice.

Reply:
People are not going to be mad at you about that (they are actually not even going to notice)

Reply:
No, but I am. That's the thing.. haha